src/App.jsx

```diff
@@ -3,42 +3,38 @@
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
+import { useState } from 'react';
+import { Tabs } from './components/Tabs/Tabs';
+
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+export const App = () => {
+  const [activeTabId, onTabSelected] = useState(tabs[0].id);
```

Comment:
The `useState` hook is used incorrectly here. The second element of the array destructuring should be a function to update the state, typically named something like `setActiveTabId`, not the event handler `onTabSelected`. You should rename `onTabSelected` to something like `setActiveTabId` and then use this function to update the state when a tab is selected.